rs/bitcoin/ckbtc/minter/tests/tests.rs

```diff
@@ -89,6 +93,19 @@ fn kyt_wasm() -> Vec<u8> {
     )
 }
 
+fn new_kyt_wasm() -> Vec<u8> {
+    load_wasm(
+        PathBuf::from(std::env::var("CARGO_MANIFEST_DIR").unwrap())
```

Comment:
Did you mean to use `IC_BTC_KYT_WASM_PATH`?

Reply:
It was just following the previous style of locating wasm files using `CARGO_MANIFEST_DIR`. Is it not recommended any more? I thought it would have the benefit of being able to run (and automatically build wasms) under cargo.